webroot/adm/enclave-gcp-v2.html

```diff
@@ -0,0 +1,191 @@
+<html>
+
+<head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+  <script src="/js/main.js"></script>
+  <style>
+    label.bold {
+      font-weight: bold
+    }
+  </style>
+</head>
+
+<body>
+<h1>UID2 Admin - GCP Enclave ID Tool(V2)</h1>
+
+<a href="/">Back</a>
+
+<br>
+<br>
+
+<h3>Inputs</h3>
+
+<label for="apiToken" class="bold">Operator Api Key:</label>
```

Comment:
We should not be collecting the API key here - we don't need it for the enclave Id, so we should not collect it. It is considered to be a secret.

Reply:
ACK, will update.